content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -161,28 +161,38 @@ Kubernetes officially supports client libraries for [Go](#go-client) and
 
 #### Go client
 
-* To get the library, run the following command: `go get k8s.io/client-go/<version number>/kubernetes` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
-* Write an application atop of the client-go clients. Note that client-go defines its own API objects, so if needed, please import API definitions from client-go rather than from the main repository, e.g., `import "k8s.io/client-go/1.4/pkg/api/v1"` is correct.
+* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-<Kubernetes version number>` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
+* Write an application atop of the client-go clients. Note that client-go defines its own API objects, so if needed, please import API definitions from client-go rather than from the main repository, e.g., `import "k8s.io/client-go/kubernetes"` is correct.
+
+{{< note >}}
+
+According to the [README](https://github.com/kubernetes/client-go/blob/master/README.md) of the client-go's responsitory, the client-go's responsitory is still a mirror of [k8s.io/kubernetes/staging/src/client-go](k8s.io/kubernetes/staging/src/client-go), the code development is still done in the staging area. Since Kubernetes 1.8 release, when syncing the code from the staging area, we also sync the Kubernetes version tags to client-go, prefixed with "kubernetes-". For example, if you check out the kubernetes-1.15.3 tag in client-go, the code you get is exactly the same as if you check out the v1.15.3 tag in Kubernetes, and change directory to staging/src/k8s.io/client-go. 
```

Comment:
Our style guide recommends against using "we". In this paragraph, I don't understand who "we" is. Can you adjust the phrasing to something like this:
Since the Kubernetes 1.8 release, the client-go library syncs ...